scipy/optimize/_direct_py.py

```diff
@@ -38,7 +34,13 @@
 
 
 def direct(
-    func: Callable[[npt.ArrayLike, tuple[Any]], float],
+    func: Callable[
+        Concatenate[
+            np.ndarray[tuple[int], np.dtype[np.float64]],
+            ...,
+        ],
+        float | np.floating[Any] | np.integer[Any] | np.bool_,
+    ],
```

Comment:
From CI:

```
E   TypeError: The last parameter to Concatenate should be a ParamSpec variable.
```

Reply:
Ah then that's only possible for `typing_extensions`... In that case, you could just replace `Concatenate[{}, ...]` with `[{}]` and remove the import. Then (asssuming the LSP is correctly implemented for `Callable` in the static type-checkers), this should allow *args and **kwargs as well.